modules/microshift-install-rhel-image-mode-run-container.adoc

```diff
@@ -0,0 +1,66 @@
+= Running the {microshift-short} bootc container
+
+Run your {microshift-short} bootc container to explore its reduced complexity and experiment with new capabilities and dependencies.
+
+.Prerequisites
+
+* A {op-system-base-full} 9.4 host with an active Red Hat subscription for building {microshift-short} bootc images and running containers.
+* You are logged into the {op-system} 9.4 host using the user credentials that have `sudo` permissions.
+* You have a pull secret file for downloading the required {microshift-short} container images.
+
+.Procedure
+
+. Run the container for the {microshift-short} service by entering the following command:
++
+[source,terminal]
+----
+PULL_SECRET=~/.pull-secret.json
+IMAGE_NAME=microshift-4.16-bootc
+
+$ sudo podman run --rm -it --privileged \
+    -v "${PULL_SECRET}":/etc/crio/openshift-pull-secret:ro \
+    -v /var/lib/containers/storage:/var/lib/containers/storage \
+    --name "${IMAGE_NAME}" \
+    "${IMAGE_NAME}"
+----
++
+[NOTE]
+====
+The `systemd-modules-load` service fails to start in the container if the host kernel version is different from the bootc image kernel version. This failure can be safely ignored as all the necessary kernel modules have been loaded by the host.
+====
+
+. A login prompt is presented in the terminal after {microshift-short} has started.
+
+. Log into the running container using the your user credentials.
+
+. Verify that all the {microshift-short} pods are up and running without errors by running the following command:
++
+[source,terminal]
+----
+$ watch sudo oc get pods -A \
+    --kubeconfig /var/lib/microshift/resources/kubeadmin/kubeconfig
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE                  NAME                                       READY   STATUS    RESTARTS      AGE
+kube-system                csi-snapshot-controller-7cfb9df49c-kc9dx   1/1     Running   0             31s
+kube-system                csi-snapshot-webhook-5c6b978878-jzk5r      1/1     Running   0             28s
+openshift-dns              dns-default-rpnlt                          2/2     Running   0             14s
+openshift-dns              node-resolver-rxvdk                        1/1     Running   0             31s
+openshift-ingress          router-default-69cd7b5545-7zcw7            1/1     Running   0             29s
+openshift-ovn-kubernetes   ovnkube-master-c7hlh                       4/4     Running   1 (16s ago)   31s
+openshift-ovn-kubernetes   ovnkube-node-mkpht                         1/1     Running   1 (17s ago)   31s
+openshift-service-ca       service-ca-5d5d96459d-5pd5s                1/1     Running   0             28s
+openshift-storage          topolvm-controller-677cbfcdb9-28dqr        5/5     Running   0             31s
+openshift-storage          topolvm-node-6fzbl                         3/3     Running   0             14s
+----
+
+. Now you can use your {microshift-short} running in the container like any {microshift-short} cluster.
```

Comment:
```suggestion
. Now you can use your {microshift-short} running in the container like any {microshift-short} cluster.
```
The IBM style mentions to use "such as" as opposed to like, but I don't think that makes sense here. Just thought I'd note it just in case.